### PR TITLE
Tried to improve the README.md, hope you like it.

### PR DESCRIPTION
**Pull request by ZigyTheBird**

### CODE REVIEW

**Comment by ZigyTheBird** [comment]
I would like [my api](https://github.com/ZigyTheBird/ZigysPlayerAnimatorAPI/wiki/Getting-Started) to be mentioned in the README.md, since it's very well documented.
But of course I will leave that up to you.

**Comment by KosmX** [comment]
Since a lot of devs would prefer server-side animation instead of low-level control, mentioning that is a good idea.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # PlayerAnimator
 
 PlayerAnimator is a minecraft library to animate the **player** while trying to break as few mods as possible.    
-If you want to add new entities, use [Geckolib](https://geckolib.com/#mods)
+If you want to add new entities, use [Geckolib](https://geckolib.com/#mods).
```
Comment (KosmX) [inline README.md:4]: You could add your API right here ^

**Comment by ZigyTheBird** [comment]
Looks good?

**Comment by KosmX** [comment]
I think I'll re-write some of this later, move some stuff into github wiki. But first 1.21.3 port.

**Pull request merged**